modules/oxide/src/less/skins/content/default/content.less

```diff
@@ -79,3 +79,12 @@ code {
     padding-right: 1rem;
   }
 }
+
+li {
+  color: black;
```

Comment:
Should this be a variable, and should this be done in all the content skins?

Reply:
no, the variables are used internally, we don't want to draw any correlation with the `content.less`. I have removed all the variable references